src/demos/_assets/demos.css

```diff
@@ -38,3 +38,55 @@ main {
   font-weight: var(--calcite-app-font-weight);
   font-size: var(--calcite-app-font-size-1);
 }
+
+/*
+  **************************************************
+  * calcite-demo homepage
+  **************************************************
+*/
+  .navigation {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 3vh 0;
+    box-sizing: border-box;
+  }
+
+  #calcite-demo {
+    color: var(--calcite-ui-brand-press);
+  }
+
+  .examples-github-storybook {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+  }
+
+  .body-margin,
+  .calcite-comp,
+  .calcite-components {
+    margin: 0 12%;
+  }
+
+  .calcite-components {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+  }
+
+  .calcite-components div {
+    flex: 0 0 25%;
+    padding: 4px 0;
+  }
+
+  .calcite-comp {
+    color: var(--calcite-ui-text-3);
+    font-family: var(--calcite-sans-family);
+    font-size: var(--calcite-font-size-0);
+    font-weight: var(--calcite-font-weight-medium);
+  }
+
+  a {
```

Comment:
Should this selector be scoped? it may mess with the style of some slotted links inside components.

Reply:
@julio8a Can you follow-up on this one? This PR should no longer blow up the build. 🚫 💥

Reply:
I'd rather @caripizza take a look so things don't 💥 again. 😁

@caripizza, can you take a look please?

Reply:
Agree with Matt, do you want me to fix that? I can just move the style to the head of the index html page if that works.

Reply:
@caripizza Yes, please. I'll hand off this PR to your capable hands. 👐🏼